spec/services/agency_seeder_spec.rb

```diff
@@ -6,10 +6,21 @@
   let(:deploy_env) { 'int' }
 
   describe '#run' do
-    before { Agency.delete_all }
+    before do
+      Agency.delete_all
+      mock_root = instance_spy(Pathname)
+      # grab the fixture file using the real Rails.root
```

Comment:
How would you feel about dropping the pair of comments here? They seem to be in the "describe what the code is doing" category instead of the "describe why the code is doing something" category.

Reply:
No strong feelings, I just tend to err on the side of overcommunication 😄. Happy to drop 'em.